lib/localparticipant.js

```diff
@@ -222,14 +222,25 @@ class LocalParticipant extends Participant {
         self.removeListener('trackDisabled', localTrackDisabled);
         self.removeListener('trackEnabled', localTrackEnabled);
         self.removeListener('trackStopped', localTrackStopped);
-
         // NOTE(mmalavalli): Remove the stale MediaTrackSender clones so that we
         // do not call replaceTrack() on their RTCRtpSenders.
         self._tracks.forEach(track => {
           const trackSignaling = localTrackStopped(track);
           if (trackSignaling) {
             track._trackSender.removeClone(trackSignaling._trackTransceiver);
           }
+
+          const trackEvents = track._events;
+          if (trackEvents) {
+            Object.keys(trackEvents).forEach(event => {
+              const emitter = trackEvents[event];
+              if (typeof emitter === 'function') {
+                track.removeListener(event, emitter);
+              } else {
+                track.removeListener(event, emitter[0]);
+              }
+            });
+          }
```

Comment:
Why is this necessary?

Reply:
I added this to remove the events on the track. However, not all of the callbacks are functions. Some appear to be an array. I'm essentially trying to get to **each** event on the track to remove them.

Reply:
Here's what it looks like 
![image](https://user-images.githubusercontent.com/43423318/99485368-2bc13100-2917-11eb-90a7-0c8bbe67fa38.png)

I was also thinking about using [removeAllListeners](https://nodejs.org/api/events.html#events_emitter_removealllisteners_eventname).

Reply:
After some debugging, we found that this is not necessary. Please remove this change.